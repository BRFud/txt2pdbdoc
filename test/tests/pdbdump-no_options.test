pdbdump | | | ISO_8859-1.pdb | 0
