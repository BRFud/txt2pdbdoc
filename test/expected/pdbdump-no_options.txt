   Name: ISO 8859-1
Version: 0
   Type: TEXt
Creator: REAd
Records: 512
===================================================================
Rec    0: [ ] Delete [ ] Dirty [ ] Busy [ ] Secret
-------------------------------------------------------------------
0000005E: 0001 0000 0000 0DD4 0001 1000 0000 0000  ................
===================================================================
Rec    1: [ ] Delete [ ] Dirty [ ] Busy [ ] Secret
-------------------------------------------------------------------
0000006E: 4465 6320 7C20 4F63 7420 7C20 4865 7820  Dec | Oct | Hex 
0000007E: 7C20 4368 7220 7C20 4368 6172 6163 7465  | Chr | Characte
0000008E: 7220 456E 7469 7479 2020 0A2D 2D2D 2D7C  r Entity  .----|
0000009E: 2D2D 2D2D 2D7C 2D2D 2D2D 2D7C 3A2D 2D2D  -----|-----|:---
000000AE: 3A7C 2D2D 2D2D 2D2D 2D2D 2D2D 2D2D 2D2D  :|--------------
000000BE: 2D2D 2D20 200A 3136 3020 7C20 3234 3020  ---  .160 | 240 
000000CE: 7C20 4130 2020 7C20 2020 2020 7C20 5C26  | A0  |     | \&
000000DE: 6E62 7370 3B20 200A 3136 3120 7C20 3234  nbsp;  .161 | 24
000000EE: 3120 7C20 4131 2020 7C20 A120 2020 7C20  1 | A1  | .   | 
000000FE: 5C26 6965 7863 6C3B 2020 0A31 3632 207C  \&iexcl;  .162 |
0000010E: 2032 3432 207C 2041 3220 207C 20A2 2020   242 | A2  | .  
0000011E: 207C 205C 2663 656E 743B 2020 0A31 3633   | \&cent;  .163
0000012E: 207C 2032 3433 207C 2041 3320 207C 20A3   | 243 | A3  | .
0000013E: 2020 207C 205C 2670 6F75 6E64 3B20 200A     | \&pound;  .
0000014E: 3136 3420 7C20 3234 3420 7C20 4134 2020  164 | 244 | A4  
0000015E: 7C20 A420 2020 7C20 5C26 6375 7272 656E  | .   | \&curren
0000016E: 3B20 200A 3136 3520 7C20 3234 3520 7C20  ;  .165 | 245 | 
0000017E: 4135 2020 7C20 A520 2020 7C20 5C26 7965  A5  | .   | \&ye
0000018E: 6E3B 2020 0A31 3636 207C 2032 3436 207C  n;  .166 | 246 |
0000019E: 2041 3620 207C 20A6 2020 207C 205C 2662   A6  | .   | \&b
000001AE: 7276 6261 723B 2020 0A31 3637 207C 2032  rvbar;  .167 | 2
000001BE: 3437 207C 2041 3720 207C 20A7 2020 207C  47 | A7  | .   |
000001CE: 205C 2673 6563 743B 2020 0A31 3638 207C   \&sect;  .168 |
000001DE: 2032 3530 207C 2041 3820 207C 20A8 2020   250 | A8  | .  
000001EE: 207C 205C 2675 6D6C 3B20 200A 3136 3920   | \&uml;  .169 
000001FE: 7C20 3235 3120 7C20 4139 2020 7C20 A920  | 251 | A9  | . 
0000020E: 2020 7C20 5C26 636F 7079 3B20 200A 3137    | \&copy;  .17
0000021E: 3020 7C20 3235 3220 7C20 4141 2020 7C20  0 | 252 | AA  | 
0000022E: AA20 2020 7C20 5C26 6F72 6466 3B20 200A  .   | \&ordf;  .
0000023E: 3137 3120 7C20 3235 3320 7C20 4142 2020  171 | 253 | AB  
0000024E: 7C20 AB20 2020 7C20 5C26 6C61 7175 6F3B  | .   | \&laquo;
0000025E: 2020 0A31 3732 207C 2032 3534 207C 2041    .172 | 254 | A
0000026E: 4320 207C 20AC 2020 207C 205C 266E 6F74  C  | .   | \&not
0000027E: 3B20 200A 3137 3320 7C20 3235 3520 7C20  ;  .173 | 255 | 
0000028E: 4144 2020 7C20 2020 2020 7C20 5C26 7368  AD  |     | \&sh
0000029E: 793B 2020 0A31 3734 207C 2032 3536 207C  y;  .174 | 256 |
000002AE: 2041 4520 207C 20AE 2020 207C 205C 2672   AE  | .   | \&r
000002BE: 6567 3B20 200A 3137 3520 7C20 3235 3720  eg;  .175 | 257 
000002CE: 7C20 4146 2020 7C20 AF20 2020 7C20 5C26  | AF  | .   | \&
000002DE: 6D61 6372 3B20 200A 3137 3620 7C20 3236  macr;  .176 | 26
000002EE: 3020 7C20 4230 2020 7C20 B020 2020 7C20  0 | B0  | .   | 
000002FE: 5C26 6465 673B 2020 0A31 3737 207C 2032  \&deg;  .177 | 2
0000030E: 3631 207C 2042 3120 207C 20B1 2020 207C  61 | B1  | .   |
0000031E: 205C 2670 6C75 736D 6E3B 2020 0A31 3738   \&plusmn;  .178
0000032E: 207C 2032 3632 207C 2042 3220 207C 20B2   | 262 | B2  | .
0000033E: 2020 207C 205C 2673 7570 323B 2020 0A31     | \&sup2;  .1
0000034E: 3739 207C 2032 3633 207C 2042 3320 207C  79 | 263 | B3  |
0000035E: 20B3 2020 207C 205C 2673 7570 333B 2020   .   | \&sup3;  
0000036E: 0A31 3830 207C 2032 3634 207C 2042 3420  .180 | 264 | B4 
0000037E: 207C 20B4 2020 207C 205C 2661 6375 7465   | .   | \&acute
0000038E: 3B20 200A 3138 3120 7C20 3236 3520 7C20  ;  .181 | 265 | 
0000039E: 4235 2020 7C20 B520 2020 7C20 5C26 6D69  B5  | .   | \&mi
000003AE: 6372 6F3B 2020 0A31 3832 207C 2032 3636  cro;  .182 | 266
000003BE: 207C 2042 3620 207C 2020 2020 207C 205C   | B6  |     | \
000003CE: 2670 6172 613B 2020 0A31 3833 207C 2032  &para;  .183 | 2
000003DE: 3637 207C 2042 3720 207C 20B7 2020 207C  67 | B7  | .   |
000003EE: 205C 266D 6964 646F 743B 2020 0A31 3834   \&middot;  .184
000003FE: 207C 2032 3730 207C 2042 3820 207C 20B8   | 270 | B8  | .
0000040E: 2020 207C 205C 2663 6564 696C 3B20 200A     | \&cedil;  .
0000041E: 3138 3520 7C20 3237 3120 7C20 4239 2020  185 | 271 | B9  
0000042E: 7C20 B920 2020 7C20 5C26 7375 7031 3B20  | .   | \&sup1; 
0000043E: 200A 3138 3620 7C20 3237 3220 7C20 4241   .186 | 272 | BA
0000044E: 2020 7C20 BA20 2020 7C20 5C26 6F72 646D    | .   | \&ordm
0000045E: 3B20 200A 3138 3720 7C20 3237 3320 7C20  ;  .187 | 273 | 
0000046E: 4242 2020 7C20 BB20 2020 7C20 5C26 7261  BB  | .   | \&ra
0000047E: 7175 6F3B 2020 0A31 3838 207C 2032 3734  quo;  .188 | 274
0000048E: 207C 2042 4320 207C 20BC 2020 207C 205C   | BC  | .   | \
0000049E: 2666 7261 6331 343B 2020 0A31 3839 207C  &frac14;  .189 |
000004AE: 2032 3735 207C 2042 4420 207C 20BD 2020   275 | BD  | .  
000004BE: 207C 205C 2666 7261 6331 323B 2020 0A31   | \&frac12;  .1
000004CE: 3930 207C 2032 3736 207C 2042 4520 207C  90 | 276 | BE  |
000004DE: 20BE 2020 207C 205C 2666 7261 6333 343B   .   | \&frac34;
000004EE: 2020 0A31 3931 207C 2032 3737 207C 2042    .191 | 277 | B
000004FE: 4620 207C 20BF 2020 207C 205C 2669 7175  F  | .   | \&iqu
0000050E: 6573 743B 2020 0A31 3932 207C 2033 3030  est;  .192 | 300
0000051E: 207C 2043 3020 207C 20C0 2020 207C 205C   | C0  | .   | \
0000052E: 2641 6772 6176 653B 2020 0A31 3933 207C  &Agrave;  .193 |
0000053E: 2033 3031 207C 2043 3120 207C 20C1 2020   301 | C1  | .  
0000054E: 207C 205C 2641 6163 7574 653B 2020 0A31   | \&Aacute;  .1
0000055E: 3934 207C 2033 3032 207C 2043 3220 207C  94 | 302 | C2  |
0000056E: 20C2 2020 207C 205C 2641 6369 7263 3B20   .   | \&Acirc; 
0000057E: 200A 3139 3520 7C20 3330 3320 7C20 4333   .195 | 303 | C3
0000058E: 2020 7C20 C320 2020 7C20 5C26 4174 696C    | .   | \&Atil
0000059E: 6465 3B20 200A 3139 3620 7C20 3330 3420  de;  .196 | 304 
000005AE: 7C20 4334 2020 7C20 C420 2020 7C20 5C26  | C4  | .   | \&
000005BE: 4175 6D6C 3B20 200A 3139 3720 7C20 3330  Auml;  .197 | 30
000005CE: 3520 7C20 4335 2020 7C20 C520 2020 7C20  5 | C5  | .   | 
000005DE: 5C26 4172 696E 673B 2020 0A31 3938 207C  \&Aring;  .198 |
000005EE: 2033 3036 207C 2043 3620 207C 20C6 2020   306 | C6  | .  
000005FE: 207C 205C 2641 456C 6967 3B20 200A 3139   | \&AElig;  .19
0000060E: 3920 7C20 3330 3720 7C20 4337 2020 7C20  9 | 307 | C7  | 
0000061E: C720 2020 7C20 5C26 4363 6564 696C 3B20  .   | \&Ccedil; 
0000062E: 200A 3230 3020 7C20 3331 3020 7C20 4338   .200 | 310 | C8
0000063E: 2020 7C20 C820 2020 7C20 5C26 4567 7261    | .   | \&Egra
0000064E: 7667 3B20 200A 3230 3120 7C20 3331 3120  vg;  .201 | 311 
0000065E: 7C20 4339 2020 7C20 C920 2020 7C20 5C26  | C9  | .   | \&
0000066E: 4561 6375 7465 3B20 200A 3230 3220 7C20  Eacute;  .202 | 
0000067E: 3331 3220 7C20 4341 2020 7C20 CA20 2020  312 | CA  | .   
0000068E: 7C20 5C26 4563 6972 633B 2020 0A32 3033  | \&Ecirc;  .203
0000069E: 207C 2033 3133 207C 2043 4220 207C 20CB   | 313 | CB  | .
000006AE: 2020 207C 205C 2645 756D 6C3B 2020 0A32     | \&Euml;  .2
000006BE: 3034 207C 2033 3134 207C 2043 4320 207C  04 | 314 | CC  |
000006CE: 20CC 2020 207C 205C 2649 6772 6176 653B   .   | \&Igrave;
000006DE: 2020 0A32 3035 207C 2033 3135 207C 2043    .205 | 315 | C
000006EE: 4420 207C 20CD 2020 207C 205C 2649 6163  D  | .   | \&Iac
000006FE: 7574 653B 2020 0A32 3036 207C 2033 3136  ute;  .206 | 316
0000070E: 207C 2043 4520 207C 20CE 2020 207C 205C   | CE  | .   | \
0000071E: 2649 6369 7263 3B20 200A 3230 3720 7C20  &Icirc;  .207 | 
0000072E: 3331 3720 7C20 4346 2020 7C20 CF20 2020  317 | CF  | .   
0000073E: 7C20 5C26 4975 6D6C 3B20 200A 3230 3820  | \&Iuml;  .208 
0000074E: 7C20 3332 3020 7C20 4430 2020 7C20 D020  | 320 | D0  | . 
0000075E: 2020 7C20 5C26 4554 483B 2020 0A32 3039    | \&ETH;  .209
0000076E: 207C 2033 3231 207C 2044 3120 207C 20D1   | 321 | D1  | .
0000077E: 2020 207C 205C 264E 7469 6C64 653B 2020     | \&Ntilde;  
0000078E: 0A32 3130 207C 2033 3232 207C 2044 3220  .210 | 322 | D2 
0000079E: 207C 20D2 2020 207C 205C 264F 6772 6176   | .   | \&Ograv
000007AE: 653B 2020 0A32 3131 207C 2033 3233 207C  e;  .211 | 323 |
000007BE: 2044 3320 207C 20D3 2020 207C 205C 264F   D3  | .   | \&O
000007CE: 6163 7574 653B 2020 0A32 3132 207C 2033  acute;  .212 | 3
000007DE: 3234 207C 2044 3420 207C 20D4 2020 207C  24 | D4  | .   |
000007EE: 205C 264F 6369 7263 3B20 200A 3231 3320   \&Ocirc;  .213 
000007FE: 7C20 3332 3520 7C20 4435 2020 7C20 D520  | 325 | D5  | . 
0000080E: 2020 7C20 5C26 4F74 696C 6465 3B20 200A    | \&Otilde;  .
0000081E: 3231 3420 7C20 3332 3620 7C20 4436 2020  214 | 326 | D6  
0000082E: 7C20 D620 2020 7C20 5C26 4F75 6D6C 3B20  | .   | \&Ouml; 
0000083E: 200A 3231 3520 7C20 3332 3720 7C20 4437   .215 | 327 | D7
0000084E: 2020 7C20 D720 2020 7C20 5C26 7469 6D65    | .   | \&time
0000085E: 733B 2020 0A32 3136 207C 2033 3330 207C  s;  .216 | 330 |
0000086E: 2044 3820 207C 20D8 2020 207C 205C 264F   D8  | .   | \&O
0000087E: 736C 6173 683B 2020 0A32 3137 207C 2033  slash;  .217 | 3
0000088E: 3331 207C 2044 3920 207C 20D9 2020 207C  31 | D9  | .   |
0000089E: 205C 2655 6772 6176 653B 2020 0A32 3138   \&Ugrave;  .218
000008AE: 207C 2033 3332 207C 2044 4120 207C 20DA   | 332 | DA  | .
000008BE: 2020 207C 205C 2655 6163 7574 653B 2020     | \&Uacute;  
000008CE: 0A32 3139 207C 2033 3333 207C 2044 4220  .219 | 333 | DB 
000008DE: 207C 20DB 2020 207C 205C 2655 6369 7263   | .   | \&Ucirc
000008EE: 3B20 200A 3232 3020 7C20 3333 3420 7C20  ;  .220 | 334 | 
000008FE: 4443 2020 7C20 DC20 2020 7C20 5C26 5575  DC  | .   | \&Uu
0000090E: 6D6C 3B20 200A 3232 3120 7C20 3333 3520  ml;  .221 | 335 
0000091E: 7C20 4444 2020 7C20 DD20 2020 7C20 5C26  | DD  | .   | \&
0000092E: 5961 6375 7465 3B20 200A 3232 3220 7C20  Yacute;  .222 | 
0000093E: 3333 3620 7C20 4445 2020 7C20 DE20 2020  336 | DE  | .   
0000094E: 7C20 5C26 5448 4F52 4E3B 2020 0A32 3233  | \&THORN;  .223
0000095E: 207C 2033 3337 207C 2044 4620 207C 20DF   | 337 | DF  | .
0000096E: 2020 207C 205C 2673 7A6C 6967 3B20 200A     | \&szlig;  .
0000097E: 3232 3420 7C20 3334 3020 7C20 4530 2020  224 | 340 | E0  
0000098E: 7C20 E020 2020 7C20 5C26 6167 7261 7665  | .   | \&agrave
0000099E: 3B20 200A 3232 3520 7C20 3334 3120 7C20  ;  .225 | 341 | 
000009AE: 4531 2020 7C20 E120 2020 7C20 5C26 6161  E1  | .   | \&aa
000009BE: 6375 7465 3B20 200A 3232 3620 7C20 3334  cute;  .226 | 34
000009CE: 3220 7C20 4532 2020 7C20 E220 2020 7C20  2 | E2  | .   | 
000009DE: 5C26 6163 6972 633B 2020 0A32 3237 207C  \&acirc;  .227 |
000009EE: 2033 3433 207C 2045 3320 207C 20E3 2020   343 | E3  | .  
000009FE: 207C 205C 2661 7469 6C64 653B 2020 0A32   | \&atilde;  .2
00000A0E: 3238 207C 2033 3434 207C 2045 3420 207C  28 | 344 | E4  |
00000A1E: 20E4 2020 207C 205C 2661 756D 6C3B 2020   .   | \&auml;  
00000A2E: 0A32 3239 207C 2033 3435 207C 2045 3520  .229 | 345 | E5 
00000A3E: 207C 20E5 2020 207C 205C 2661 7269 6E67   | .   | \&aring
00000A4E: 3B20 200A 3233 3020 7C20 3334 3620 7C20  ;  .230 | 346 | 
00000A5E: 4536 2020 7C20 E620 2020 7C20 5C26 6165  E6  | .   | \&ae
00000A6E: 6C69 673B 2020 0A32 3331 207C 2033 3437  lig;  .231 | 347
00000A7E: 207C 2045 3720 207C 20E7 2020 207C 205C   | E7  | .   | \
00000A8E: 2663 6365 6469 6C3B 2020 0A32 3332 207C  &ccedil;  .232 |
00000A9E: 2033 3530 207C 2045 3820 207C 20E8 2020   350 | E8  | .  
00000AAE: 207C 205C 2665 6772 6176 653B 2020 0A32   | \&egrave;  .2
00000ABE: 3333 207C 2033 3531 207C 2045 3920 207C  33 | 351 | E9  |
00000ACE: 20E9 2020 207C 205C 2665 6163 7574 653B   .   | \&eacute;
00000ADE: 2020 0A32 3334 207C 2033 3532 207C 2045    .234 | 352 | E
00000AEE: 4120 207C 20EA 2020 207C 205C 2665 6369  A  | .   | \&eci
00000AFE: 7263 3B20 200A 3233 3520 7C20 3335 3320  rc;  .235 | 353 
00000B0E: 7C20 4542 2020 7C20 EB20 2020 7C20 5C26  | EB  | .   | \&
00000B1E: 6575 6D6C 3B20 200A 3233 3620 7C20 3335  euml;  .236 | 35
00000B2E: 3420 7C20 4543 2020 7C20 EC20 2020 7C20  4 | EC  | .   | 
00000B3E: 5C26 6967 7261 7665 3B20 200A 3233 3720  \&igrave;  .237 
00000B4E: 7C20 3335 3520 7C20 4544 2020 7C20 ED20  | 355 | ED  | . 
00000B5E: 2020 7C20 5C26 6961 6375 7465 3B20 200A    | \&iacute;  .
00000B6E: 3233 3820 7C20 3335 3620 7C20 4545 2020  238 | 356 | EE  
00000B7E: 7C20 EE20 2020 7C20 5C26 6963 6972 633B  | .   | \&icirc;
00000B8E: 2020 0A32 3339 207C 2033 3537 207C 2045    .239 | 357 | E
00000B9E: 4620 207C 20EF 2020 207C 205C 2669 756D  F  | .   | \&ium
00000BAE: 6C3B 2020 0A32 3430 207C 2033 3630 207C  l;  .240 | 360 |
00000BBE: 2046 3020 207C 20F0 2020 207C 205C 2665   F0  | .   | \&e
00000BCE: 7468 3B20 200A 3234 3120 7C20 3336 3120  th;  .241 | 361 
00000BDE: 7C20 4631 2020 7C20 F120 2020 7C20 5C26  | F1  | .   | \&
00000BEE: 6E74 696C 6465 3B20 200A 3234 3220 7C20  ntilde;  .242 | 
00000BFE: 3336 3220 7C20 4632 2020 7C20 F220 2020  362 | F2  | .   
00000C0E: 7C20 5C26 6F67 7261 7665 3B20 200A 3234  | \&ograve;  .24
00000C1E: 3320 7C20 3336 3320 7C20 4633 2020 7C20  3 | 363 | F3  | 
00000C2E: F320 2020 7C20 5C26 6F61 6375 7465 3B20  .   | \&oacute; 
00000C3E: 200A 3234 3420 7C20 3336 3420 7C20 4634   .244 | 364 | F4
00000C4E: 2020 7C20 F420 2020 7C20 5C26 6F63 6972    | .   | \&ocir
00000C5E: 633B 2020 0A32 3435 207C 2033 3635 207C  c;  .245 | 365 |
00000C6E: 2046 3520 207C 20F5 2020 207C 205C 266F   F5  | .   | \&o
00000C7E: 7469 6C64 653B 2020 0A32 3436 207C 2033  tilde;  .246 | 3
00000C8E: 3636 207C 2046 3620 207C 20F6 2020 207C  66 | F6  | .   |
00000C9E: 205C 266F 756D 6C3B 2020 0A32 3437 207C   \&ouml;  .247 |
00000CAE: 2033 3637 207C 2046 3720 207C 20F7 2020   367 | F7  | .  
00000CBE: 207C 205C 2664 6976 6964 653B 2020 0A32   | \&divide;  .2
00000CCE: 3438 207C 2033 3730 207C 2046 3820 207C  48 | 370 | F8  |
00000CDE: 20F8 2020 207C 205C 266F 736C 6173 683B   .   | \&oslash;
00000CEE: 2020 0A32 3439 207C 2033 3731 207C 2046    .249 | 371 | F
00000CFE: 3920 207C 20F9 2020 207C 205C 2675 6772  9  | .   | \&ugr
00000D0E: 6176 653B 2020 0A32 3530 207C 2033 3732  ave;  .250 | 372
00000D1E: 207C 2046 4120 207C 20FA 2020 207C 205C   | FA  | .   | \
00000D2E: 2675 6163 7574 653B 2020 0A32 3531 207C  &uacute;  .251 |
00000D3E: 2033 3733 207C 2046 4220 207C 20FB 2020   373 | FB  | .  
00000D4E: 207C 205C 2675 6369 7263 3B20 200A 3235   | \&ucirc;  .25
00000D5E: 3220 7C20 3337 3420 7C20 4643 2020 7C20  2 | 374 | FC  | 
00000D6E: FC20 2020 7C20 5C26 7575 6D6C 3B20 200A  .   | \&uuml;  .
00000D7E: 3235 3320 7C20 3337 3520 7C20 4644 2020  253 | 375 | FD  
00000D8E: 7C20 FD20 2020 7C20 5C26 7961 6375 7465  | .   | \&yacute
00000D9E: 3B20 200A 3235 3420 7C20 3337 3620 7C20  ;  .254 | 376 | 
00000DAE: 4645 2020 7C20 FE20 2020 7C20 5C26 7468  FE  | .   | \&th
00000DBE: 6F72 6E3B 2020 0A32 3535 207C 2033 3737  orn;  .255 | 377
00000DCE: 207C 2046 4620 207C 20FF 2020 207C 205C   | FF  | .   | \
00000DDE: 2679 756D 6C3B 0A                        &yuml;.
